logic.py

```diff
@@ -995,7 +995,7 @@ def diff(y, x):
 
 
 def simp(x):
-    "Simplify the expression x."
+    """Simplify the expression x."""
```

Comment:
I'm actually not a stickler for triple quotes on short doc strings. But the PEP does suggest triple quotes, and all your other work was very helpful, so I'll agree to this.